hs/Database/Database.hs

```diff
@@ -28,7 +29,8 @@ setupDatabase = do setupDistributionTable
                    print "breadth table set up"
                    parseAll
                    seedVideos
-
+                   getDepartments
+                   
```

Comment:
The whitespace still appears to be in the file. Perhaps you didn't commit the change.